packages/webapp/components/FooterNavBar.tsx

```diff
@@ -83,19 +83,20 @@ export default function FooterNavBar({
       ) : (
         <ScrollToTopButton />
       )}
-      <Flipper
-        flipKey={activeTab}
-        spring="veryGentle"
-        element="nav"
-        className={classNames(
-          'grid w-full grid-flow-col items-center justify-between rounded-16 px-3',
-          !post && activeClasses,
-          !showNav && 'hidden',
-          !post && 'border-t border-border-subtlest-tertiary',
-        )}
-      >
-        <FooterNavBarTabs activeTab={activeTab} />
-      </Flipper>
+      {showNav && (
```

Comment:
We don't need to render the footer navbar on tablet and desktop which is evaluated at `showNav`.